### PR TITLE
Personal keymap for the planck with muzak

### PR DESCRIPTION
**Pull request by sidcarter**
Personal keymap for the planck with muzak

## Description

* Has Colemak on a layer
* Enables mouse layer, but does not activate it yet
* Has Muzak and Sounds (Wheeee)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/planck/keymaps/skank/keymap.c [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+ * | Tab  |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  | Bksp |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * | Esc  |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   ;  |  "   |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * | Shift|   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |  Up  |Enter |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Ctrl | Alt  | GUI  |      |Lower | Enter|Space |Raise |   /  | Left | Down |Right |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_QWERTY] = {
```
Comment (drashna) [inline keyboards/planck/keymaps/skank/keymap.c]: You may want to (probably should) convert these to the `LAYOUT_ortho_4x12` style, as seen in the default keymap.

File: keyboards/planck/keymaps/skank/keymap.c [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+[_ADJUST] = {
+  {_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______},
+  {_______, _______, _______, AU_ON,   AU_OFF,   QWERTY, COLEMAK, _______, _______, _______, _______,   RESET},
+  {_______, MUV_DE,  MUV_IN,  MU_ON,   MU_OFF,   MI_ON,   MI_OFF, _______, _______, _______, _______, _______},
+  {_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______}
+}
+
+
+};
+#ifdef AUDIO_ENABLE
```
Comment (drashna) [inline keyboards/planck/keymaps/skank/keymap.c]: This block isn't being used, and can be removed.

Comment (sidcarter) [reply keyboards/planck/keymaps/skank/keymap.c]: remove that block

File: keyboards/planck/keymaps/skank/keymap.c (hunk 1 above)
Comment (sidcarter) [reply keyboards/planck/keymaps/skank/keymap.c]: converted to the style as seen in the default keymap.

**Pull request merged**